.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
why do we need both the file `.babelrc` and `babel.config.js`? Can we delete `.babelrc`?

Reply:
yes, I think so. I will delete it.